Python/Tests/Core/EnvironmentListTests.cs

```diff
@@ -945,7 +945,12 @@ public class EnvironmentListTests {
                     python.Version.ToVersion()
                 )
             );
-            service.AddPackageManagers(factory, new CPythonPipPackageManagerProvider(null).GetPackageManagers(factory).ToList());
+            var pmList = new CPythonPipPackageManagerProvider(null)
+                .GetPackageManagers(factory)
+                .Where(p => p != null)
+                .ToList();
+
+            service.AddPackageManagers(factory, pmList);
```

Comment:
adding null filters

Reply:
Both package manager providers in this PR check and make sure they don't yield return null, so there's no need to check for that AFAIK. If they don't have package managers, they'll return empty enumerable, not an enumerable with a null entry.

Reply:
Ah I see that now.